sensor_msgs/PointCloud2[] point_clouds
